be/src/item/item.service.ts

```diff
@@ -0,0 +1,49 @@
+import { Injectable } from '@nestjs/common';
+import { InjectRepository } from '@nestjs/typeorm';
+import { Connection, Repository } from 'typeorm';
+import { CreateItemInput } from './dto/create-item.input';
+import { UpdateItemInput } from './dto/update-item.input';
+import { Item } from './entities/item.entity';
+
+@Injectable()
+export class ItemService {
+  constructor(
+    @InjectRepository(Item) private readonly itemRepository: Repository<Item>,
+    private readonly connection: Connection,
+  ) {}
+
+  createItem(createItemInput: CreateItemInput): Promise<Item> {
+    const newItem = this.itemRepository.create(createItemInput);
+
+    return this.itemRepository.save(newItem);
+  }
+
+  findAll() {
+    return this.itemRepository.find({});
+  }
+
+  findOne(id: number) {
+    return this.itemRepository.findOne(id);
```

Comment:
Please check whether it was found and if not return a better error than `Cannot return null for non-nullable field Query.item.`.
Also do this for notes

```gql
query test {
  # If item #2200 doesn't exists
  # the code will return a confusing err
  item(id: 2200) {
    id
  }
}
```